frontend/index.tsx

```diff
@@ -120,6 +121,39 @@ const Relic = () => {
         }
     }
 
+    function createFilesTable(metaDatas: MetaData[]) {
+        if (metaDatas.length === 0)
+            return "";
+        else {
+            return (
+                <>
+                    <Table variant="simple">
+                        <Thead>
+                            <Tr>
+                                {Object.keys(metaDatas[0]).map(key => (
+                                    <Th>{key}</Th>
+                                ))}
+                            </Tr>
+                        </Thead>
+                        <Tbody>
+                            {metaDatas.map(metaData => (
+                                <Tr>
+                                    {Object.keys(metaData).map(key => (
+                                        <Td>{metaData[key as keyof MetaData]}</Td>
```

Comment:
i didnt even know about thei keyof thing!

Reply:
All credit goes to J.Pat